### ISSUE TITLE
Recording multiple errors and re-using attribute names.

### ISSUE THREAD

**Issue by martinjuckes** [cp4cds/c3s_34g_qc_results#1]
The first draft appears to support only a single error per file with `cf_error_code` and `cf_status` provided as strings. Can we modify this to, for example `cf_error_codes` giving a list of error codes and `cf_warning_code` giving a list of warnings?

**Comment by stephank16** [comment]
yes, there are also other tags where we probably need to allow lists as values - let's first collect and review examples - then I will review and provide a json-schema formalizing our defined structure ..
